src/main/java/com/google/sps/data/backgroundTaskManagerInterface.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+import java.util.Optional;
+import java.util.List;
+
+/** Interface for the backgroundTaskManager class. */
+public interface backgroundTaskManagerInterface { 
```

Comment:
Class/Interface names should be UpperCamelCase

https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names

Reply:
Marking unresolved, this needs to be capitalized.